scripts/test

```diff
@@ -85,4 +90,11 @@ if [ -f "/tmp/vim-go-test/FAILED" ]; then
 fi
 echo 2>&1 "All tests PASSED"
 
+# Submit coverage reports
+if [ -n "$coverage" ]; then
+  coverage xml --omit '*_test.vim'
+  codecov -X search gcov pycov -f coverage.xml --required --flags "$vim"
```

Comment:
You need to replace things, i.e. `codecov_flags=${vim//,/}; codecov_flags=${codecov_flags//-/}` and then use this wiht `--flags $codecov_flags`.

Reply:
Hm, I replace `-` and `.` now (sending `vim80` instead of `vim-8.0`) and still doesn't seem to work. Not sure if this is a Codecov error. I've noticed that codecov can behave quirky if there is no report for master yet (for example, it keeps showing "Reports are queued for processing... Please review report with caution, it may change." for this PR).

Reply:
The replacement does not seem to work:

```
.query pr=1586&service=travis&package=py2.0.9&job=308161852&flags=vim-7.4&build=621.1&branch=master&commit=c26daeaa606e0abafadfe3efae3895b8bd898d11&slug=fatih%2Fvim-go
```

Reply:
..it uses `&flags=vim-7.4`.

Reply:
`build=621.1` is the previous build though. For some reason Travis never started a build for the latest commit (which also explains why it doesn't show in Codecov). I pushed another commit and now it seems to work 👍